### ISSUE TITLE
Any examples of automated restoration tasks from BBR backup?

### ISSUE THREAD

**Issue by aegershman** [pivotal-cf/bbr-pcf-pipeline-tasks#36]
Although restoring from a BBR backup can be a rather complicated process, I have not seen any examples of automation workflows which restore any of these artifacts that are being backed up (director, PAS, PKS, or any BBR artifact really).

Are there existing examples of using Concourse to restore from a BBR backup?

Thanks for your consideration

**Comment by aegershman** [comment]
Closing this out, but would still be interested to see if/how other orgs automate their BBR restores / what their restoration strategy is in general. Thanks!

**Comment by alamages** [comment]
Hello @aegershman 

Sorry for not updating you earlier. We have automated the backup procedure, to help users take frequent backups. Restore is a procedure that only happens in case of a disaster (data corruption, disk lost etc), hopefully, this is infrequent. We have decided to not automate the restore process and let the users decide when to restore based on their best practice and policies in place. For this reason, there does not exist a concourse pipeline and tasks example on restoring.

Thanks,
Emmanouil & @MModhaPivotal
